### PR TITLE
Clean up unused functions / iex load

### PR DESCRIPTION
**Pull request by jmaslek**
Address #1042  and #1044 

@Chavithra not sure if this will destroy some tests

- Removes the two unused plot functions
- Redefines the iex loading and eliminates intraday for non yf data

### CODE REVIEW

**Review by Chavithra — CHANGES_REQUESTED** [review]
Apparently it can't find `s_start` variable in a print statement.

Here is the code I have run :
```python
from datetime import datetime
from gamestonk_terminal.stocks import stocks_helper

stock = stocks_helper.load(
    ticker="TSLA",
    start=datetime.strptime("2021-12-05", "%Y-%m-%d"),
    interval=1440,
)
```

Here is the displayed message :
```
local variable 's_start' referenced before assignment
```

File: gamestonk_terminal/stocks/stocks_helper.py [hunk 1]
```diff
@@ -94,6 +93,7 @@ def load(
     end: datetime = datetime.now(),
     prepost: bool = False,
     source: str = "yf",
+    iexrange: str = "tyd",
```
Comment (Chavithra) [inline gamestonk_terminal/stocks/stocks_helper.py:96]: I think it should be "ytd"

**Comment by jmaslek** [comment]
> Apparently it can't find `s_start` variable in a print statement.
> 
> Here is the code I have run :
> 
> ```python
> from datetime import datetime
> from gamestonk_terminal.stocks import stocks_helper
> 
> stock = stocks_helper.load(
>     ticker="TSLA",
>     start=datetime.strptime("2021-12-05", "%Y-%m-%d"),
>     interval=1440,
> )
> ```
> 
> Here is the displayed message :
> 
> ```
> local variable 's_start' referenced before assignment
> ```

Fixed.  Had an incorrect tab

**Review by Chavithra — APPROVED** [review]
Seems to work, was able to :
- pass the test with `iex daily`
- get `candle graph` for `iex loaded`data`

File: gamestonk_terminal/stocks/stocks_controller.py [hunk 2]
```diff
@@ -284,7 +284,7 @@ def call_load(self, other_args: List[str]):
             "--source",
             action="store",
             dest="source",
-            choices=["yf", "av", "iex"],
+            choices=["yf", "av", "iex"] if "-i" not in other_args else ["yf"],
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/stocks_controller.py:287]: nice one

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**